README.md

```diff
@@ -168,15 +160,11 @@ Flutter ile ilgili Türkçe makaleler kategorilerine göre ayrılmış halde bur
 ##### DSC Edirne
 - [Flutter vs React Native](https://youtu.be/BtxsOehgl_s) - Sercan Yusuf & Özge Aydın
 
+<!--
 ##### Developer MultiGroup
```

Comment:
Can you update this section from this playlist?

https://www.youtube.com/watch?v=lVVcrBnGzEA&list=PLZj2Q42tw-68zEBhpO_2mwX-1roiSvEHd

Reply:
I will do it.